Public/Sdk/Public/Managed/Tools/NuGet/pack.dsc

```diff
@@ -125,9 +125,10 @@ export function pack(args: Arguments): PackResult {
 
     let execArgs = <Transformer.ExecuteArguments>{
         tool: args.tool || tool,
-        tags: (args.tags || ["package"] ),
+        tags: (args.tags || ["package"]),
         arguments: arguments,
         workingDirectory: outDir,
+        allowUndeclaredSourceReads: Context.getCurrentHost().os === "macOS",

```

Comment:
>allowUndeclaredSourceReads [](start = 8, length = 26)

too much to track mono safely?

Reply:
Indeed, the actual worker (Nuget) is tracked, Mono just rages all over the place as the child it is executing (Nuget) does all those file accesses. We still depend on system folders and files, so when updates and env changes, we invalidate and repackage, so we do on source file changes so this should be enough.